include/villas/kernel/if.hpp

```diff
@@ -48,7 +48,7 @@ class Interface {
 	int affinity;				// IRQ / Core Affinity of this interface.
 
 	std::list<int> irqs;			// List of IRQs of the NIC.
-	std::list<node::Node *> nodes;	// List of nodes which use this interface.
+	std::list<node::Node *> nodes;	        // List of nodes which use this interface.
```

Comment:
My suggestion was using spaces instead of tabs for indentation. My bad.

Reply:
Fixed :)